app/controllers/idv_controller.rb

```diff
@@ -39,11 +39,7 @@ def sp_over_quota_limit?
 
   def verify_identity
     analytics.track_event(Analytics::IDV_INTRO_VISIT)
-    if proof_with_cac?
-      redirect_to idv_cac_url
-    else
-      redirect_to idv_doc_auth_url
-    end
+    redirect_to idv_doc_auth_url
   end
```

Comment:
this seems like not a useful method, if it just logs an analytics event and redirects, can we
- remove it
- go to the source and update it to link to `idv_doc_auth_url`
- update `idv_doc_auth_url` to log the event?

Reply:
It's usefulness is in the context of being the single endpoint/decider for all future forms of document verification.  It had two options until we removed one but the endpoint is still reliable and can be used when we add passports, etc.